app/src/main/java/com/yml/womensafety/navigationdrawer/TipsForWomenSafetyFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.yml.womensafety.navigationdrawer
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.yml.womensafety.TipsForWomenSafetyAdapter
+import com.yml.womensafety.R
+
+class TipsForWomenSafetyFragment : Fragment(R.layout.fragment_tips_for_women_safety) {
+
+    private val tipsTextList = ArrayList<SafetyTips>()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val recyclerView = view.findViewById(R.id.recyclerView) as RecyclerView
+        recyclerView.layoutManager = LinearLayoutManager(view.context)
+
+
+        displaySafetyTips()
+
+        val adapter = TipsForWomenSafetyAdapter(tipsTextList)
+        recyclerView.adapter = adapter
+    }
+
+    private fun displaySafetyTips() {
+        tipsTextList.add(SafetyTips("If walking by yourself, get off your cell phone. Chatting can distract you, " +
+                "it's better to be aware of your surroundings."))
+        tipsTextList.add(SafetyTips("If walking by yourself, hold your cell phone and be ready to make an emergency call."))
+        tipsTextList.add(SafetyTips("Look underneath your car before approaching it in a parking garage or lot at night."))
```

Comment:
Try using 
```suggestion
        tipsTextList.apply{
        add(SafetyTips("Look underneath your car before approaching it in a parking garage or lot at night."))
        add(SafetyTips("If you return to your car and see that a van is parked right next to the driver's side, " +
                "enter through the passenger side. Predators often use vans and will disguise it as a family car."))
                .
                .
                .
        }
```
Likewise

Reply:
Done